src/store/modules/news.store.js

```diff
@@ -0,0 +1,20 @@
+import categoryStateInit from "./state.init";
+
+export default {
+	namespaced: true,
+	state: {
+		category: categoryStateInit("aktualnosci", 2, 1),
```

Comment:
do we want to have names in Polish?

Reply:
both polish and english depends on the lang - it is slug id in wordpress.